mq/main/bridge/bridge-api/src/main/java/com/sun/messaging/bridge/api/StompFrameMessageFactory.java

```diff
@@ -22,4 +22,8 @@
 public interface StompFrameMessageFactory {
 
     StompFrameMessage newStompFrameMessage(StompFrameMessage.Command cmd, LoggerWrapper logger);
+
+    void setVersion(String version);
```

Comment:
It doesn't feel right place for version. Factory does not use this, and only serves as transport for it. I'd need (more) time to investigate other possibilities (or less time to review other solution).